README.md

```diff
@@ -3634,4 +3634,4 @@ The project was created having Philip Wadler's talk ["A practical theory of lang
 
 * [A Practical Theory of Language-Integrated Query](http://homepages.inf.ed.ac.uk/slindley/papers/practical-theory-of-linq.pdf)
 * [Everything old is new again: Quoted Domain Specific Languages](http://homepages.inf.ed.ac.uk/wadler/papers/qdsl/qdsl.pdf)
-* [The Flatter, the Better](http://db.inf.uni-tuebingen.de/staticfiles/publications/the-flatter-the-better.pdf)
\ No newline at end of file
+* [The Flatter, the Better](http://db.inf.uni-tuebingen.de/staticfiles/publications/the-flatter-the-better.pdf)
```

Comment:
I didn't mean to change this line, I don't know why it got introduced. We may take it out.

Reply:
I undid this with a second commit, it was probably just a newline.